packages/client/base/src/types/blockchain.ts

```diff
@@ -1,17 +1,35 @@
 import { UIConfig } from "./uiconfig";
 
-export const EVM_CHAINS = ["ethereum", "polygon", "bsc"] as const;
-export const ALL_CHAINS = ["solana", "cardano", ...EVM_CHAINS] as const;
-export type EVMChain = (typeof EVM_CHAINS)[number];
+export const EVM_CHAINS = [
+    "ethereum",
+    "ethereum-sepolia",
+    "polygon",
+    "bsc",
+    "optimism",
+    "arbitrum",
+    "base",
+    "zora",
+    "arbitrumnova",
+    "zkatana",
+] as const;
+export const BLOCKCHAIN_TEST_NET = ["goerli", "ethereum-sepolia", "mumbai", "zkatana"] as const;
+export const ALL_CHAINS = ["solana", "cardano", "sui", ...EVM_CHAINS] as const;
+export const EVM_BLOCKCHAIN_INCLUDING_TESTNET = [...EVM_CHAINS, ...BLOCKCHAIN_TEST_NET] as const;
+export const BLOCKCHAIN_INCLUDING_TESTNET = [...ALL_CHAINS, ...BLOCKCHAIN_TEST_NET] as const;
+
 export type Blockchain = (typeof ALL_CHAINS)[number];
+export type EVMBlockchain = (typeof EVM_CHAINS)[number];
```

Comment:
this is EVMChain normally, right? Did we rename because there's already an EVMChain in the repo?

Reply:
No, just to be consistent with the other namings